client/app/private-database/configuration/private-database-configuration.html

```diff
@@ -1,76 +1,67 @@
 <div class="container-fluid px-0" data-ng-controller="PrivateDatabaseConfigurationsCtrl as configurationsCtrl">
 
+    <div data-ovh-alert="{{alerts.main}}"></div>
+
     <div class="text-center" data-ng-if="configurationsCtrl.loading">
```

Comment:
It should be a ui-kit class to be reused across all managers. An align attribute on oui-spinner could be a possibility, it's a recurrent requirement.

Reply:
Having `text-center` as a ui-kit class is a nice idea but an align attribute would be against W3C and WHATWG specs

Reply:
I'm curious, why it's against W3C and WHATWG?

Reply:
According to @antleblanc I'm wrong.

I just assumed defining alignment outside of CSS would be viewed as wrong, as HTML's align attribute (https://www.w3.org/TR/html401/present/graphics.html#h-15.1.2) is deprecated, but according to him we can do whatever we want with components without having to worry with standards, I really don't know :smiley:

Reply:
I agree "defining alignment outside of CSS would be viewed as wrong"
IMHO "text-center" class has the same problem..
I could be better a more semantic class name e.g.: oui-page__content-loader without alignment concern
We will keep text-center as a temporary solution..